pages/index.tsx

```diff
@@ -1,154 +1,86 @@
-import Head from 'next/head'
+// import { activateAndSetCookie } from '../utils/i18n'
+// import dayjs from 'dayjs'
+// import { useTimezone } from '../store/useTimezone'
+
 import {
   Button,
-  Checkbox,
-  FormControl,
-  FormControlLabel,
-  FormHelperText,
-  FormLabel,
-  Radio,
-  RadioGroup,
-} from '@material-ui/core'
-import styles from '../styles/Home.module.css'
-import { activateAndSetCookie } from '../utils/i18n'
+  Container,
+  Typography,
+  Box,
+  useMediaQuery,
+  useTheme,
+} from '@mui/material'
+import React from 'react'
 import { Trans } from '@lingui/macro'
-import dayjs from 'dayjs'
-import { useTimezone } from '../store/useTimezone'
-
-export default function Home(): JSX.Element {
-  const localTimezone = useTimezone((state) => state.localTimezone)
-  const defaultTimezone = useTimezone((state) => state.defaultTimezone)
-  const setTimezone = useTimezone((state) => state.setTimezone)
-  const handleTimeZone = (e: any) => {
-    setTimezone(e.target.value)
-  }
+import FrontPage from '../src/components/templates/FrontPage'
+import fetcher from '../src/utils/fetcher'
 
-  return (
-    <div className={styles.container}>
-      <Head>
-        <title>Create Next App</title>
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
+export default function Index({ events, upcomingEvents, groups }): JSX.Element {
+  // const localTimezone = useTimezone((state) => state.localTimezone)
+  // const defaultTimezone = useTimezone((state) => state.defaultTimezone)
+  // const setTimezone = useTimezone((state) => state.setTimezone)
+  // const handleTimeZone = (e: any) => {
+  //   setTimezone(e.target.value)
+  // }
```

Comment:
We need to enable this for DateTime localization. Let's follow up on this in next PR.